docs/architecture/decisions/0003-where-to-publish-container-image.md

```diff
@@ -0,0 +1,34 @@
+# 1. Where to publish the container image
+
+Date: 2022-09-07
+
+## Status
+
+Proposed
+
+## Context
+
+This project is packaged as a container image. Choose the container registry where deliver the image.
+
+### Option 1
+
+Publish the image to [GitHub Packages](https://docs.github.com/en/packages/learn-github-packages/introduction-to-github-packages).
+
+- Cost: The repository is public, so the GitHub Packages is [Free](https://docs.github.com/en/billing/managing-billing-for-github-packages/about-billing-for-github-packages#about-billing-for-github-packages).
+- Effort: GitHub Packages is fully integrated with GitHub.
+- Limits: There are no know limits about GitHub Packages.
```

Comment:
Can you have anonymous access to the Github container (like when using dockerhub) or you must be logged into Github?

Reply:
Public image allow anonymous access. Check [this](https://docs.github.com/en/packages/learn-github-packages/about-permissions-for-github-packages#:~:text=Public%20images%20allow%20anonymous%20access%20and%20can%20be%20pulled%20without%20authentication%20or%20signing%20in%20via%20the%20CLI.) out.

Reply:
Thanks guys, I've add this information 👍